Sources/Danger/BitBucketCloud.swift

```diff
@@ -0,0 +1,228 @@
+import Foundation
+
+public struct BitBucketCloud: Decodable {
+    /// The activities such as OPENING, CLOSING, MERGING or UPDATING a pull request
+    public let activities: [BitBucketCloudActivity]
+
+    /// The comments on the pull request
+    public let comments: [BitBucketCloudComment]
+
+    /// The commits associated with the pull request
+    public let commits: [BitBucketCloudCommit]
+
+    /// The pull request and repository metadata
+    public let metadata: BitBucketMetadata
+
+    /// The PR metadata
+    public let pr: BitBucketCloudPR
```

Comment:
<!--
  0 failure: 
  1 warning:  Variable name sho...
  
  
  DangerID: danger-id-Danger;
  File: Sources/Danger/BitBucketCloud.swift;
  Line: 17;
-->

- :warning: Variable name should be between 3 and 40 characters long: 'pr' (`identifier_name`)